index.d.ts

```diff
@@ -53,6 +53,7 @@ export interface AxiosRequestConfig {
   adapter?: AxiosAdapter;
   auth?: AxiosBasicCredentials;
   responseType?: ResponseType;
+  responseEncoding?: string;
```

Comment:
I think you could define a ResponseEncoding type as

```javascript
export type ResponseEncoding =
  | "ascii"
  | "utf8"
  | "utf-8"
  | "utf16le"
  | "ucs2"
  | "ucs-2"
  | "base64"
  | "latin1"
  | "binary"
  | "hex"
```

and then define the optional config even better:

```suggestion
  responseEncoding?: ResponseEncoding;
```

Reply:
I took this from the @types/node globals.d.ts BufferEncoding type.

Reply:
@Magmagan Thanks for your tips. But I'd like keep with [Node.js API documents](https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end) temporarily and not limit to some certain types.